benefits/static/css/styles.css

```diff
@@ -395,6 +395,7 @@ footer .footer-links li a.footer-link:visited {
   outline: 3px solid var(--focus-color) !important;
   outline-offset: 2px !important;
   box-shadow: none !important; /* override CA State Web Template */
+  text-decoration: none;
```

Comment:
Just wanted to leave a comment here for everyone else: This is necessary to override a Django Admin CSS style for buttons. Without this, there would be an underline on the button.